app/src/main/java/ar/com/wolox/android/cookbook/googleLogin/GoogleLoginPresenter.kt

```diff
@@ -0,0 +1,22 @@
+package ar.com.wolox.android.cookbook.googleLogin
+
+import android.content.Intent
+import ar.com.wolox.android.cookbook.googleLogin.model.UserGoogle
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class GoogleLoginPresenter @Inject constructor() : BasePresenter<IGoogleLoginView>() {
+
+    fun onGoogleLogin(data: Intent?) {
```

Comment:
Remember that from the `Presenter` we don't reference Android specific code because that makes it difficult to test. And the reason for that is: if we have an Android dependency, we have to mount lots of the Android Framework when testing.

We should create another object that hides this mechanism internally and allows us to fetch the `GoogleAccount`.